pkgs/flox-cli/default.nix

```diff
@@ -120,18 +117,16 @@
     # runtime dependencies of the dependent crates
     buildInputs =
       [
-        openssl.dev # octokit -> hyper -> ssl
-        zlib # git2
-        libssh2 # git2
-        libgit2 # git2
+        # reqwest -> hyper -> openssl-sys
+        openssl.dev
       ]
       ++ lib.optional hostPlatform.isDarwin [
-        darwin.apple_sdk.frameworks.Security # git2 (and others)
+        darwin.libiconv
```

Comment:
Curious about this change, but not blocking as the tests pass.

Reply:
libiconv seems to have come together with `libgit2`
at least removing `libgit2` required adding `libiconv` explictly